### PR TITLE
[BZ1447862] Cloud Networks 

### PR DESCRIPTION
**Pull request by cbudz**
Hey @suyogsainkar 

I've added a new section related to cloud network features. I'm waitng on a some feedback, but if you'd like to start taking a look at what I've got here, that would be great. 
 
A preview can be found in [BZ1447862](https://bugzilla.redhat.com/show_bug.cgi?id=1447862).

Please feel free to let me know if you have any questions.

-Chris

### CODE REVIEW

File: doc-Managing_Infrastructure_and_Inventory/topics/Adding_a_Cloud_Network.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[[adding-a-cloud-network]]
+
+=== Adding a Cloud Network
+
+
+Add a new cloud network following the steps in this procedure:
+
+. Navigate to menu:Networks > Networks. 
+. Click image:1847.png(Configuration) and click *Add a new Cloud Network*.
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Adding_a_Cloud_Network.adoc]: The navigation menu and the image syntax needs an edit for it to render correctly.

File: doc-Managing_Infrastructure_and_Inventory/topics/Networks.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+== Creating and Administering Cloud Networks
+
+Create a network to provide instances a place to communicate internally and receive IP addresses using Dynamic Host Configuration Protocol (DHCP). A network can also be integrated with external networks in your Red Hat OpenStack Platform deployment or elsewhere, such as the physical network.  
+
+
+[NOTE]
+====
+* Keystone API v3 is required to create cloud tenants on Red Hat OpenStack Platform cloud providers. For more information, see https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/11/html-single/architecture_guide/#comp-identity[OpenStack Identity (keystone)] in the Red Hat OpenStack Platform _Architecture Guide_.
+* Please see the link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/12/html-single/networking_guide/#add_an_interface[_Networking Guide_] in the Red Hat OpenStack Platform documentation for further details on cloud network objects and administration.
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Networks.adoc]: It's okay to start without please when directing users to other documentation: See the Networking Guide..

File: doc-Managing_Infrastructure_and_Inventory/topics/Adding_an_Interface_to_Network_Router.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[[adding-interface-network-routers]]
+=== Adding an Interface to a Network Router  
+
+Interfaces allow you to interconnect routers with subnets. As a result, the router can direct any traffic that instances send to destinations outside of their intermediate subnet.
+
+To add an interface to a network router:
+
+. Navigate to menu:Networks[Network Routers].
+. Select a network router from the list view. 
+. Click  image:1847.png[Configuration] (*Configuration*), then  image:1851.png[Add Interface to selected Router] (*Add Interface to selected Router*).
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Adding_an_Interface_to_Network_Router.adoc]: I had a quick look at the 6 Feb build, UI shows "Add Interface to this Router". Also, as per convention, we avoid using drop-down menu and either just say "from the list" or "from the *Subnet* list* - worth bringing this up in our meeting for a quick discussion if required.

File: doc-Managing_Infrastructure_and_Inventory/topics/Deleting_a_Router.adoc [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[[delete-router]]
+
+=== Deleting a Network Router
+
+To delete a network router:
+
+. Navigate to menu:Networks[Network Routers].
+. Select a network router from the list view. 
+. Click image:1847.png[Configuration](*Configuration*), then click *Delete selected Routers.
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Deleting_a_Router.adoc]: Rendering issue with Delete selected Routers - minor syntax error.

**Comment by cbudz** [comment]
Hey @suyogsainkar 

Thanks for catching those items. I've made corrections and updated the preview. 

-Chris

**Comment by suyogsainkar** [comment]
Thanks Chris. Merging to master.

**Pull request merged**